test/functional/buildAndPackage/playlist.xml

```diff
@@ -14,6 +14,16 @@
 -->
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>Java_Version</testCaseName>
+		<command>$(JAVA_COMMAND) -version; $(TEST_STATUS)</command>
```

Comment:
Please add `$(JVM_OPTIONS)`. It will prevent any problem in case we add variation in the future.

Reply:
If new variation is added should separate specific command line tests be added?

Reply:
No, if it is just adding new variations, there is no need to add separate tests. Please see https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/openjdk/playlist.xml#L19